manifests/server.pp

```diff
@@ -21,7 +21,7 @@
     if $::puppet::server_git_repo {
       $config_version_cmd = "git --git-dir ${::puppet::server_envs_dir}/\$environment/.git describe --all --long"
     } else {
-      $config_version_cmd = ''
+      $config_version_cmd = undef
```

Comment:
I was considering the same and even wrote some code to play with it. I strongly considered adding all template parameters and get rid of ```scope.lookupvar```. Would that be something you agree on?